source/includes/api/_widget.md

```diff
@@ -1,428 +1,296 @@
+page[number] |       The page number        | Number | 1
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?dataset=d02df2f6-d80c-4274-bb6f-f062061655c4
-curl -X GET https://api.resourcewatch.org/v1/dataset/d02df2f6-d80c-4274-bb6f-f062061655c4/widget
-```
+### Filters
 
-> Sort by name
+> Return the widgets filtered by those whose name contains emissions
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?sort=name
+curl -X GET "https://api.resourcewatch.org/v1/widget?name=emissions"
 ```
 
-> Filter widgets by published status
+> Return the widgets filtered by dataset
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?published=false
+curl -X GET "https://api.resourcewatch.org/v1/widget?dataset=11de2bc1-368b-42ed-a207-aaff8ece752b"
+curl -X GET "https://api.resourcewatch.org/v1/dataset/11de2bc1-368b-42ed-a207-aaff8ece752b/widget"
 ```
 
-> Filter widgets by verified status
+> Filter widgets by default value
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?verified=false
+curl -X GET "https://api.resourcewatch.org/v1/widget?default=false"
 ```
 
-> Return the widgets filtered by template
+> Filter widgets by environment
+
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?template=true
+curl -X GET "https://api.resourcewatch.org/v1/widget?env=staging"
 ```
 
-> Filter widgets by default option
+> Return the widgets filtered by those whose applications contain both `rw` and `prep` applications simultaneously
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?default=true
+curl -X GET "https://api.resourcewatch.org/v1/widget?app=rw@prep"
 ```
 
-> Return widgets whose applications contain rw
+The widget list endpoints provide a wide range of filters that you can use to tailor your widget listing. The great majority of the widget fields you'll find on the [widget reference](#widget-reference) section can be used as filters when listing widgets, with the following exceptions:
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?app=rw
-```
+- `id`
+- `userName`
+- `userRole`: filtering by the role of the owning user can be done using the `user.role` query argument.
 
-### Pagination params
+Additionally, you can use the following filters:
 
-Field        |       Description        |   Type
------------- | :----------------------: | -----:
-page[size]   | Number elements per page. There's a limit of 100 to the size of each page, which is not being enforced at the moment, but queries for larger page sizes are not supported. This means future requests may fail if not respecting the page size limit. | Number
-page[number] |      Number of page      | Number
+- `collection`: filters by a [collection](#collections) id. Requires being authenticated.
+- `favourite`: if defined, only the widgets set by the user as [favorites](#favorites) will be returned. Requires being authenticated.
```

Comment:
Is this a boolean filter? (if so, that should be in the description of the filter)

Reply:
I'll clarify the language, thanks